posthog-core/src/types.ts

```diff
@@ -124,9 +128,8 @@ export type PostHogAutocaptureElement = {
   nth_child?: number
   nth_of_type?: number
   order?: number
-} & {
-  [key: string]: any
-} // Any key prefixed with `attr__` can be added
+} & PostHogEventProperties
+// Any key prefixed with `attr__` can be added
```

Comment:
this comment is interesting, isn't it

there's nothing stopping us adding keys which aren't prefixed with `attr__` but that could be enforced by types here

not blocking on this PR just not seen that comment before

Reply:
yeah comes from RN autocapture

Reply:
so could have

```
type AttrPrefixed = {
  [key: `attr__${string}`]: JsonType
}
```